.github/workflows.src/nightly.tpl.yml

```diff
@@ -193,7 +193,7 @@ jobs:
         printf 'FROM scratch\nENV EDGEDB_USER edgedb\nADD edgedb /usr/bin/\nENTRYPOINT ["/usr/bin/edgedb"]\n' \
           >dockerctx/Dockerfile
         tag=$(date +%Y%m%d%H%M%S)$(echo "${GITHUB_SHA}" | cut -c1-6)
-        echo ::set-env name=SNAPSHOT_TAG::${tag}
+        echo "SNAPSHOT_TAG=${tag}" >> $GITHUB_ENV
```

Comment:
See previous conversation in this PR. This version is preferred, the other is deprecated as insecure. The rendered workflow files were already updated.